procurement.md

```diff
@@ -0,0 +1,695 @@
+is somehow unsuited for government use.  Rather, government has not
+yet spent a decade honing open source practices that take into account
+the particular needs of government software development.
+
+One place this disparity appears is in contracting.  Government has an
+obligation to safeguard the public trust.  Its operation is accountable
+to public political process.  Public investments are judged by
+different metrics than those used in private-sector commerce.  While government has
+much to gain from open source and is in the process of developing and
+spreading best practices on how to do that, it cannot simply borrow
+procurement models wholesale from the private sector.
+
+That is not to say that government lacks the knowledge or ability to
+succeed at FOSS projects.  It does suggest, though, that
+approaches should be designed with care for the particular constraints
+and opportunities found in public agencies.
+
+## Modular Contracting
+
+One center of excellence in government development of FOSS software is
+18F, the digital service delivery arm of the United States General
+Services Adminisration.  They focus on technology procurement for
+federal agencies, and their methodology typically begins with agile
+and open source approaches.  18F is generally considered a model
+agency when it comes to open source.  They advise federal projects to
+consider procurement based on a model of "modular contracting," (See
+https://18f.gsa.gov/tags/modular-contracting/) in which large projects
+are procured via a set of small contracts, each to the vendor most
+suited for a particular task.  This is a good starting point for state
+and local technology procurement as well. (18F addresses non-federal
+modular contracting at
+https://18f.gsa.gov/2016/11/15/modular-procurement-state-local-government/.)
+
+This approach has several benefits, beginning most importantly with
+breaking vendor lock-in.  At any point, a well-procured effort should
+provide recourse to multiple, credible vendors, each of whom has
+familiarity with the software, experience working with the other
+teams, and is able to work well with the agency's open source
+approach.  Every vendor becomes replaceable because none by itself is
+so crucial to the process that it cannot be replaced.
+
+With that goal in mind, 18F and others in the industry recommend
+structuring procurement as multiple contracts distributed among a
+group of vendors.  They also recommend relatively small contracts,
+though "small" often refers to a different scale, depending on the
+locale and the level of government.
+
+It is difficult to make one-size-fits-all recommendations as to the
+correct size of a modular contract.  That will depend on the overall
+budget, the project, and the ecosystem needs at a particular stage of
+development.  For a medium-sized informational website project, one
+might employ contracts no smaller than $60,000.  This size allows
```

Comment:
Providing numbers is as useful as it is difficult. So I am very supportive of providing numbers like this one. I wonder if we would need to contextualize this number to another reference number to account for geographic differences. Does this $60k mean the same to a government official in the US than it does to their counterpart in Sierra Leone? And I bring this up because the intended audience of this text is more the latter than the former, so I want to ensure it is meaningful to them. Curious to hear your thoughts on this.

Reply:
We essentially need to use a reference currency throughout the document, and state it up front, and maybe (near the front) give a few examples of what a given reference amount would be in some specific pathfinder countries.

I'm not sure that that reference currency should be USD or EU, but I'm also not sure what to choose instead.  Any advice?